arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,552 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-05-06",
+    "title": "AzureAnalysisServices",
+    "description": "The Azure Analysis Services Web API provides a RESTful set of web services that enables users to create, retrieve, update, and delete servers",
+    "x-ms-code-generation-settings": {
+      "name": "AzureAnalysisServicesClient"
+    }
+  },
+  "host": "management.azure.com",
+  "basePath": "/webapi",
+  "schemes": [ "https" ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{analysisServerName}": {
+      "get": {
+        "description": "Gets the current state of the specified Analysis Services server",
+        "tags": [ "RpProvision" ],
+        "operationId": "RpProvision_GetDetailsAsync",
+        "parameters": [
+          {
+            "name": "analysisServerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Name of the Analysis Services server"
+          },
+          {
+            "name": "resourceGroupName",
```

Comment:
It's a good idea to have common parameters like `resourceGroupName` defined in the global parameters section. However, since it should be provided for each method that uses it as a parameter, you can use the [`x-ms-parameter-location` extension](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-extensions.md#x-ms-parameter-location) so that it isn't a parameter on the service client.

That way there's no duplication in the spec